core/commands/follow.go

```diff
@@ -0,0 +1,182 @@
+package commands
+
+import (
+	"errors"
+	"io"
+	"strings"
+
+	cmds "github.com/ipfs/go-ipfs/commands"
+	e "github.com/ipfs/go-ipfs/core/commands/e"
+
+	"gx/ipfs/QmUyfy4QSr3NXym4etEiRyxBLqqAeKHJuRdi8AACxg63fZ/go-ipfs-cmdkit"
+)
+
+type ipnsFollowResult struct {
+	OK bool
```

Comment:
maybe instead of a boolean, we just have a `State` field thats a string, since you convert it into a string later anyways

Reply:
We generally use `SetError` to indicate that the command has failed anyways. We should probably do something like what we do in `ipfs pin add`: `{"Follows": [CIDs]}`.

Reply:
Not sure it adds much though -- in reality it's just a void command that can error.

Reply:
Actually, I think it's best to just use an empty struct for void and just output ok.
As @Stebalien points out, we indicate errors with SetError anyway.

Reply:
Hrm, can't have it be just void. I will make it a string as why suggested.